.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Linux
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - run: git remote -v
+    - name: Fetch branches
+      run: |
+        git config remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
+        git fetch --unshallow
+    - run: git branch -avv
+    - run: git log --oneline --graph --decorate -20
+    - run: ./bootstrap
+    - run: ./configure
+    - run: make
```

Comment:
This line isn't needed. The next line does the `make` as well.

Reply:
I know that.
Every step in a job is listed as pass or fail.
If the `make` already fails one can see that with a quick glance at the job.
If it's done in one step one doesn't know yet if the testsuite failed or already the `make`.
Is there any advantage in removing the line?

Reply:
That makes sense. It's fine to leave it in.